rules/gdi-vehicles.yaml

```diff
@@ -1107,3 +1121,81 @@ IONTANK:
 		Voice: Move
 		Weight: 20
 	HitShape:
+
+MARV:
+	Inherits@1: ^VoxelTank
+	Inherits@2: ^SuperHeavyVehicle
+	Inherits@FeatureAutotargetVehicle: ^FeatureAutotargetVehicle1
+	Inherits@FeatureAutotargetAircraft: ^FeatureAutotargetAircraft1
+	Inherits@FeatureAutotargetInfantry: ^FeatureAutotargetInfantry2
+	Inherits@FeatureAutotargetBuilding: ^FeatureAutotargetBuilding3
+	Valued:
+		Cost: 8500
+	Tooltip:
+		Name: M.A.R.V.
+	Buildable:
+		Queue: Vehicle.GDI
+		BuildPaletteOrder: 127
+		BuildLimit: 1
+		Prerequisites: ~gaweap, gatech, ~!t1, ~!t2, ~upgrade.zc
+		Description: Zocom's ultra-heavy tank.\n\nGood vs: Ground\n\nSpecial:\n- Self Healing\n- Can destroy Tiberium\n- Crush class: Super Heavy vehicle
+	Selectable:
+		Bounds: 42,42, 0, -5
+	Voiced:
+		VoiceSet: Marv
+		Volume: 2
+	Mobile:
+		Speed: 45
+		TurnSpeed: 3
+	Health:
+		HP: 6500
+	RevealsShroud:
+		Range: 14c745
+	AutoTarget:
+	Turreted:
+		TurnSpeed: 2
+		Offset: -500,0,0
+	Explodes:
+		Weapon: UnitExplodeEpic
+		EmptyWeapon: UnitExplodeEpic
+	Harvester:
+		DeliveryBuildings: kahpad
```

Comment:
kahpad? I guess you wouldn't like marv being able to throw resources only to cabal's refinery, insert all of them